### ISSUE TITLE
Issue with multipal choices in model interactions (bug?)

### ISSUE THREAD

**Issue by Mothball4242** [darktrovx/interact#9]
Hi guys,

So since the recent updates, adding multipal options and individual model interaction causes it to not show correctly. With previous updates we could just add multipal interactions to the same model and they would show as a list. Now the list doesn't how and bugs out (see image).

https://i.imgur.com/iIwb8qX.png

There also seem to be an issue where even if a model has only a single option it still whoes with the multioption pip to the left even though it didnt in the previous updates?

Thanks, love the script guys.

**Comment by FjamZoo** [comment]
Could you provide the code that you are using for multiple points showing up on the same model?

**Comment by Mothball4242** [comment]
Hey, sorry for th delay getitng back.

Here is the code we are using to add items to a vending machine. The old version currently works fine.

`local QBCore = exports['qb-core']:GetCoreObject() 

local src = source
local snack_items1 = 'sandwich'
local snack_items2 = 'twerks_candy'
local snack_items3 = 'snikkel_candy'
local snack_items4 = 'tosti'

exports.interact:AddModelInteraction({

    modelData = {
        { model = 'prop_vend_snak_01', offset = vec3(0.0, 0.0, 0.0) },
        { model = 'prop_vend_snak_01_tu', offset = vec3(0.0, 0.0, 0.0) },
    },
    --name = 'interactionName', -- optional
    distance = 8.0, -- optional
    interactDst = 3.0, -- optional
    options = {
        {
            label = 'Water Bottle ($10)',
            action = function(entity, coords, args)
                TriggerServerEvent('geo-vending:server:attemptPurchase', snack_items1) 
            end,
        },
    }
})

exports.interact:AddModelInteraction({

    modelData = {
        { model = 'prop_vend_snak_01', offset = vec3(0.0, 0.0, 0.0) },
        { model = 'prop_vend_snak_01_tu', offset = vec3(0.0, 0.0, 0.0) },
    },
    --name = 'interactionName', -- optional
    distance = 8.0, -- optional
    interactDst = 3.0, -- optional
    options = {
        {
            label = 'Water Bottle ($10)',
            action = function(entity, coords, args)
                TriggerServerEvent('geo-vending:server:attemptPurchase', snack_items2) 
            end,
        },
    }
})

exports.interact:AddModelInteraction({

    modelData = {
        { model = 'prop_vend_snak_01', offset = vec3(0.0, 0.0, 0.0) },
        { model = 'prop_vend_snak_01_tu', offset = vec3(0.0, 0.0, 0.0) },
    },
    --name = 'interactionName', -- optional
    distance = 8.0, -- optional
    interactDst = 3.0, -- optional
    options = {
        {
            label = 'Water Bottle ($10)',
            action = function(entity, coords, args)
                TriggerServerEvent('geo-vending:server:attemptPurchase', snack_items3) 
            end,
        },
    }
})`

I have also tried just using "model = 'prop_vend_snak_01'," individually like in your updated documentation.

Cheers!

**Comment by FjamZoo** [comment]
Yeah make sense you would need to just add the 3 options for the models like this

```lua
local models = {
    `prop_vend_snak_01`,
    `prop_vend_snak_01_tu`
}

for i = 1, #models do
    exports.interact:AddModelInteraction({
        model = models[i],
        distance = 8.0,
        interactDst = 3.0,
        options = {
            {
                label = 'Water Bottle ($10)',
                action = function(entity, coords, args)
                    TriggerServerEvent('geo-vending:server:attemptPurchase', snack_items1)
                end,
            },
            {
                label = 'Water Bottle ($10)',
                action = function(entity, coords, args)
                    TriggerServerEvent('geo-vending:server:attemptPurchase', snack_items2)
                end,
            },
            {
                label = 'Water Bottle ($10)',
                action = function(entity, coords, args)
                    TriggerServerEvent('geo-vending:server:attemptPurchase', snack_items3) 
                end,
            },
        }
    })
end
```
